chia/cmds/wallet.py

```diff
@@ -479,9 +497,14 @@ def make_offer_cmd(
     filepath: str,
     fee: str,
     reuse: bool,
+    no_confirm: bool,
+    no_file: bool,
 ) -> None:
     from .wallet_funcs import make_offer
 
+    if filepath == "" and not no_file:
```

Comment:
is the offer printed to stdout when `--no-file` is passed?
There's a convention among some unix tools to use `-` as the filename to indicate `stdout` or `stdin`

Reply:
no its just saved, but not outputted at all.

Reply:
i could remove that arg i guess.

Reply:
so `--no-file` is also a pure test option then, right? It would never be useful in production. If so, it would seem like a better solution to have the tests output to temporary directories

Reply:
removed